### PR TITLE
postcss: Mockup simple test to setup NODE_PATH and PATH

### PR DESCRIPTION
**Pull request by richtera**
I was playing around with jsBuild and took some of the path iteration stuff
to allow testing this theory.

This is to play with #7656

### CODE REVIEW

File: resources/resource_transformers/postcss/postcss.go [hunk 1]
```diff
@@ -140,69 +143,88 @@ func (t *postcssTransformation) Key() internal.ResourceTransformationKey {
...
-		if err != nil {
-			if t.options.Config != "" {
-				// Only fail if the user specificed config file is not found.
-				return errors.Wrapf(err, "postcss config %q not found:", configFile)
+	nodePath := ""
+	binPath := ""
+	for idx, mod := range t.rs.PathSpec.Paths.AllModules {
+		dir := mod.Dir()
+		if localCommand == "" {
+			commandPath := path.Join(dir, "node_modules", ".bin", binaryName)
```
Comment (bep) [inline resources/resource_transformers/postcss/postcss.go:162]: So, the idea is good, but this assumes that `npm install` has been run -- which Hugo cannot do as a general thing (modules pulled in via Hugo Modules are read-only)

Comment (richtera) [reply resources/resource_transformers/postcss/postcss.go:162]: Interesting. That would assume that npm folders need to either be published as part of the module, or updated in some other way. It does make sense, but that will make a lot of things more difficult if modules' js files require things which are not necessarily available on the project node_modules and package.json. That's in interesting dilemma :)

**Comment by bep** [comment]
I'm closing this.

**Comment by richtera** [comment]
Ok, thanks

**Pull request closed without merging**